swarm/tools/flow_studio_ui/css/flow-studio.base.css

```
@@ -1064,6 +1064,33 @@
+      font-size: 16px;
+      cursor: pointer;
+      padding: 0;
+      line-height: 1;
+      width: 20px;
+      height: 20px;
+      border-radius: 50%;
+      display: none;
+      align-items: center;
+      justify-content: center;
+      transition: color 0.15s, background 0.15s;
+    }
```

Stylelint (17.2.0)

[error] Expected empty line before rule (rule-empty-line-before)

(rule-empty-line-before)